.github/actions/build-ab/index.js

```diff
@@ -133,3 +134,68 @@ if (['dev', 'staging'].includes(args.environment)) {
     }
   }
 }
+
+// add entitlements NG script shim
+(async () => {
+  if (environment === 'dev') return
+  const entitlementsEndpoints = {
+    dev: '',
+    staging: 'https://staging-one.newrelic.com',
+    prod: 'https://one.newrelic.com',
+    'eu-prod': 'https://one.eu.newrelic.com'
+  }
+  const entitlementsNG = `
+  try {
```

Comment:
So I was considering using a templating library before when I was converting the original scripts to a github action. I decided against it because there was no code that needed to be injected. Since that is changing, maybe we should consider EJS or Handlebars.

Reply:
Yea that would be cool. I have no preference to what we use. Just something easy!!!